vita3k/modules/SceGxm/SceGxm.cpp

```diff
@@ -124,19 +124,21 @@ EXPORT(int, sceGxmBeginCommandList, SceGxmContext *deferredContext) {
 }
 
 EXPORT(int, sceGxmBeginScene, SceGxmContext *context, uint32_t flags, const SceGxmRenderTarget *renderTarget, const SceGxmValidRegion *validRegion, SceGxmSyncObject *vertexSyncObject, Ptr<SceGxmSyncObject> fragmentSyncObject, const SceGxmColorSurface *colorSurface, const SceGxmDepthStencilSurface *depthStencil) {
-    assert(flags == 0);
-    assert(renderTarget);
-    assert(vertexSyncObject == nullptr);
-
     if (!context) {
         return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
     }
+
+    if (flags & 0xFFFFFFF0) {
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_VALUE);
+    }
+
+    if (!renderTarget || (vertexSyncObject != nullptr)) {
+        return RET_ERROR(SCE_GXM_ERROR_INVALID_POINTER);
```

Comment:
value return "SCE_GXM_ERROR_INVALID_POINTER" already writed for context, you an add check inside same ?

Reply:
No as this will not respect the order of the checks

Reply:
oh i see :)